### PR TITLE
greencoffeettc api v3

### PR DESCRIPTION
**Pull request by markphi2013**

### CODE REVIEW

**Comment by overnin** [comment]
@markphi2013 i reviewed already, please fix my comments

**Comment by markphi2013** [comment]
@overnin  i don't see any comments

**Comment by markphi2013** [comment]
@overnin  i haved the comments 
- [ ]  moved responses for separate json files
- [ ]  removed commented code 
- [ ] added user message content to the test

File: transports/http_forward/greencoffeelizard_v3_http.py [hunk 1]
```diff
@@ -153,24 +153,24 @@ def handle_outbound_message(self, message):
...
             else:
                 keyword_sent = message['content'].split(' ', 1)[0]
-                keywords_inuse = [self.config['yes_agent_prices_keyword'],
-                                  self.config['yes_company_prices_keyword'],
-                                  self.config['yes_weather_keyword']]
+                keywords_inuse = [self.config.get('yes_agent_prices_keyword'),
+                                  self.config.get('yes_company_prices_keyword'),
+                                  self.config.get('yes_weather_keyword')]
                 if keyword_sent in keywords_inuse:
                     event_contents = self.build_location_code_url_response(response_body['results'], keyword_sent)
```
Comment (markphi2013) [inline transports/http_forward/greencoffeelizard_v3_http.py]: @overnin i need some  logic help here
if user sends in for coffee price 
```event_content = {'AgentPrice': {u'max': 45700, u'timestamp': 1501200000000, u'min': 45700}}```

I would like the participant to get message **"Agent price *[timestamp]* *[max]* "**

else user sends in for weather info 
```{'WindDir': {u'max': 235, u'timestamp': 1501502400000, u'min': 235}, 'MinTemp': {u'max': 21, u'timestamp': 1501502400000, u'min': 21}, 'WindSpeed': {u'max': 2, u'timestamp': 1501502400000, u'min': 2}, 'MaxTemp': {u'max': 27, u'timestamp': 1501502400000, u'min': 27}, 'Precipitation': {u'max': 7, u'timestamp': 1501502400000, u'min': 7}}```

i would like the participant to get **" weather *[cod1]* *[cond1Max]* at  *[cod1Timestamp]*, *[cond2]*  [cond2Max] at *[cond2Timestamp]*, .......... "**

NB: cond == condition (WindDir, MinTemp, WindSpeed,... )
How can i generate such cos iam abit stack?

**Pull request closed without merging**